unit_test/live_model_test.cc

```diff
@@ -1945,8 +1945,6 @@ BOOST_AUTO_TEST_CASE(live_model_using_endpoint_failure_no_uri)
   BOOST_CHECK_EQUAL(_rl->init(&status), r::error_code::http_model_uri_not_provided);
 }
 
```

Comment:
USE_AZURE_FACTORIES is defaulted to ON so it won't make a difference in the CI but maybe it should be added back in case someone needs to compile without them?

Reply:
yeah good catch - let me fx it

Reply:
actually this flag was in random place - most of the tests in this file are having this dependency as well.
Created an issue: https://github.com/VowpalWabbit/reinforcement_learning/issues/560